src/components/AudioButton/AudioButton.test.js

```diff
@@ -0,0 +1,29 @@
+import AudioButton from './AudioButton'
+import { render } from '@testing-library/react'
+
+describe('AudioButton', () => {
+  it('renders correctly', () => {
+    const { container } = render(
+      <AudioButton
+        songUrl={'https://www.testurl.com'}
+        id={'nj8se0eqez237'}
+        currentSong={'nj8se0eqez237'}
+        setCurrentSong={() => {}}
+      />
+    )
+    expect(container.firstChild).toMatchSnapshot()
+  })
+
+  it('shows play button per default', () => {
+    const { isSongPlaying, getByTitle } = render(
+      <AudioButton
+        songUrl={'https://www.testurl.com'}
+        id={'nj8se0eqez237'}
+        currentSong={'nj8se0eqez237'}
+        setCurrentSong={() => {}}
+      />
+    )
+    expect(isSongPlaying).toBeFalsy()
+    expect(getByTitle('play')).toBeInTheDocument()
```

Comment:
Aaaah, is that's why the button icons have titles? Another idea for a test would be that after a click on the icon, stuff changes. Like: 

```
const button = getByRole('button')
user.click(button)
expect(isSongPlaying).toBeTruthy()
expect(getByTitle('pause')).toBeInTheDocument()
expect(getByTitle('play')).not.toBeInTheDocument()
```

Maybe for the last line it must be `queryByTitle` (if that even exists :) because it might be the case that otherwise it'll throw an error, if the test cant find anything with the title "play" (as it's not in rendered anymore). I dont know. You'll figure s.th. out :))